diff -aurN ams-1.8.6.orig/modularsynth.cpp ams-1.8.6/modularsynth.cpp
--- ams-1.8.6.orig/modularsynth.cpp	2004-07-16 14:44:03.000000000 +0200
+++ ams-1.8.6/modularsynth.cpp	2004-07-23 18:52:53.662374632 +0200
@@ -258,8 +258,8 @@
                      "\nby Matthias Nagorni and Fons Adriaensen\n"
                      "(c)2002-2003 SuSE AG Nuremberg\n"
                      "(c)2003 Fons Adriaensen\n\n" 
-                     "Documentation and examples can be found in\n"
-                     "/usr/share/doc/packages/kalsatools\n\n"
+                     "Examples can be found in\n"
+                     "/usr/share/ams\n\n"
                      "More presets and updates are available from\n"
                      "http://alsamodular.sourceforge.net"
     "\n\nAcknowledgements\n"
