diff -aurN ams-1.5.9.orig/make_ams ams-1.5.9/make_ams
--- ams-1.5.9.orig/make_ams	2003-05-15 19:29:24.000000000 +0200
+++ ams-1.5.9/make_ams	2003-06-19 16:02:12.000000000 +0200
@@ -1,5 +1,5 @@
-QT_BASE_DIR=/usr/lib/qt3
-QT_LIB_DIR=$(QT_BASE_DIR)/lib
+QT_BASE_DIR=/usr
+QT_LIB_DIR=$(QT_BASE_DIR)/lib/qt
 QT_BIN_DIR=$(QT_BASE_DIR)/bin
 QT_INCLUDE_DIR=$(QT_BASE_DIR)/include
 X11_LIB_DIR=/usr/X11R6/lib
@@ -114,8 +114,6 @@
 	-L$(QT_LIB_DIR) -L$(X11_LIB_DIR) \
 	-lqt-mt -ljack -lasound -lsrfftw -lsfftw -lm 
 
-CXXFLAGS=-DQT_THREAD_SUPPORT -I$(QT_INCLUDE_DIR) -I/usr/X11R6/include -I. -O2 -g -Wall
-
 synth.o: synth.cpp synth.h synthdata.h module.h m_out.h m_wavout.h m_scope.h m_spectrum.h m_midiout.h
 capture.o: capture.cpp capture.h synthdata.h m_in.h m_jackout.h m_jackin.h
 synthdata.o: synthdata.cpp synthdata.h module.h port.h m_env.h m_advenv.h
