diff -aurN ams-1.7.2.orig/Makefile ams-1.7.2/Makefile
--- ams-1.7.2.orig/Makefile	2004-01-20 11:53:27.000000000 +0100
+++ ams-1.7.2/Makefile	2004-01-23 17:30:40.321727648 +0100
@@ -1,13 +1,13 @@
-QT_BASE_DIR=/usr/lib/qt3
-QT_LIB_DIR=$(QT_BASE_DIR)/lib
+QT_BASE_DIR=/usr
+QT_LIB_DIR=$(QT_BASE_DIR)/lib/qt
 QT_BIN_DIR=$(QT_BASE_DIR)/bin
-QT_INCLUDE_DIR=$(QT_BASE_DIR)/include
+QT_INCLUDE_DIR=$(QT_BASE_DIR)/include/qt
 X11_LIB_DIR=/usr/X11R6/lib
 LADSPA_PATH?=/usr/lib/ladspa:/usr/local/lib/ladspa
 VERSION=1.7.2
 
 #CXXFLAGS=-DQT_THREAD_SUPPORT -I$(QT_INCLUDE_DIR) -DLADSPA_PATH=\"$(LADSPA_PATH)\" -I/usr/X11R6/include -I. -O2 -g -Wall
-CXXFLAGS=-DQT_THREAD_SUPPORT -I$(QT_INCLUDE_DIR) -DLADSPA_PATH=\"$(LADSPA_PATH)\" -I/usr/X11R6/include -I. -O2 -Wall
+CXXFLAGS=-DQT_THREAD_SUPPORT -I$(QT_INCLUDE_DIR) -DLADSPA_PATH=\"$(LADSPA_PATH)\" -I/usr/X11R6/include -I. $(OPTFLAGS) -Wall 
 
 AMS_O = synthdata.o module.o module.moc.o \
 	configdialog.o configdialog.moc.o \
@@ -70,9 +70,9 @@
 	main.o alsa_driver.o
 
 ams:	$(AMS_O)
-	gcc -g -o ams $(AMS_O) \
+	$(CC) -g -o ams $(AMS_O) \
 	-L$(QT_LIB_DIR) -L$(X11_LIB_DIR) \
-	-lqt-mt -ljack -lasound -lsrfftw -lsfftw -lm
+	-lqt-mt -ljack -lasound -lrfftw -lfftw -lm
 
 clean:
 	/bin/rm -f *.o *.moc.cpp *~
