diff -aurN ams-1.8.7.orig/Makefile ams-1.8.7/Makefile
--- ams-1.8.7.orig/Makefile	2004-07-21 10:50:43.000000000 +0200
+++ ams-1.8.7/Makefile	2005-06-26 18:40:36.000000000 +0200
@@ -1,13 +1,13 @@
-QT_BASE_DIR=/usr/lib/qt3
-QT_LIB_DIR=$(QT_BASE_DIR)/lib
+QT_BASE_DIR=/usr
+QT_LIB_DIR=$(QT_BASE_DIR)/lib/qt
 QT_BIN_DIR=$(QT_BASE_DIR)/bin
-QT_INCLUDE_DIR=$(QT_BASE_DIR)/include
+QT_INCLUDE_DIR=$(QT_BASE_DIR)/include/qt
 X11_LIB_DIR=/usr/X11R6/lib
 LADSPA_PATH?=/usr/lib/ladspa:/usr/local/lib/ladspa
 VERSION=1.8.6
 
 #CXXFLAGS=-DQT_THREAD_SUPPORT -I$(QT_INCLUDE_DIR) -DLADSPA_PATH=\"$(LADSPA_PATH)\" -I/usr/X11R6/include -I. -O2 -g -Wall
-CXXFLAGS=-DQT_THREAD_SUPPORT -I$(QT_INCLUDE_DIR) -DLADSPA_PATH=\"$(LADSPA_PATH)\" -I/usr/X11R6/include -I. -O2 -Wall
+CXXFLAGS=-DQT_THREAD_SUPPORT -I$(QT_INCLUDE_DIR) -DLADSPA_PATH=\"$(LADSPA_PATH)\" -I/usr/X11R6/include -I. $(OPTFLAGS) -Wall
 
 AMS_O = synthdata.o module.o module.moc.o \
 	configdialog.o configdialog.moc.o \
