diff -aurN ams-1.8.6.orig/Makefile ams-1.8.6/Makefile
--- ams-1.8.6.orig/Makefile	2004-07-16 14:44:03.000000000 +0200
+++ ams-1.8.6/Makefile	2004-07-23 18:49:44.033202648 +0200
@@ -1,13 +1,13 @@
-QT_BASE_DIR=/usr/lib/qt3
-QT_LIB_DIR=$(QT_BASE_DIR)/lib
+QT_BASE_DIR=/usr
+QT_LIB_DIR=$(QT_BASE_DIR)/lib/qt
 QT_BIN_DIR=$(QT_BASE_DIR)/bin
-QT_INCLUDE_DIR=$(QT_BASE_DIR)/include
+QT_INCLUDE_DIR=$(QT_BASE_DIR)/include/qt
 X11_LIB_DIR=/usr/X11R6/lib
 LADSPA_PATH?=/usr/lib/ladspa:/usr/local/lib/ladspa
 VERSION=1.8.6
 
 #CXXFLAGS=-DQT_THREAD_SUPPORT -I$(QT_INCLUDE_DIR) -DLADSPA_PATH=\"$(LADSPA_PATH)\" -I/usr/X11R6/include -I. -O2 -g -Wall
-CXXFLAGS=-DQT_THREAD_SUPPORT -I$(QT_INCLUDE_DIR) -DLADSPA_PATH=\"$(LADSPA_PATH)\" -I/usr/X11R6/include -I. -O2 -Wall
+CXXFLAGS=-DQT_THREAD_SUPPORT -I$(QT_INCLUDE_DIR) -DLADSPA_PATH=\"$(LADSPA_PATH)\" -I/usr/X11R6/include -I. $(OPTFLAGS) -Wall
 
 AMS_O = synthdata.o module.o module.moc.o \
 	configdialog.o configdialog.moc.o \
@@ -80,7 +80,7 @@
 ams:	$(AMS_O)
 	gcc -g -o ams $(AMS_O) \
 	-L$(QT_LIB_DIR) -L$(X11_LIB_DIR) \
-	-lqt-mt -ljack -lasound -lsrfftw -lsfftw -lm
+	-lqt-mt -ljack -lasound -lrfftw -lfftw -lm
 
 clean:
 	/bin/rm -f *.o *.moc.cpp *~
